--- templates/horizon_local_settings.py.j2	2018-05-01 10:19:07.000000000 -0500
+++ templates/horizon_local_settings.py.j2	2018-05-21 16:27:44.000000000 -0500
@@ -176,7 +176,7 @@
 WEBSSO_ENABLED = True

 # Determines which authentication choice to show as default.
-WEBSSO_INITIAL_CHOICE = "credentials"
+WEBSSO_INITIAL_CHOICE = "saml2"

 # The list of authentication mechanisms which include keystone
 # federation protocols and identity provider/federation protocol
@@ -247,6 +247,18 @@
 LAUNCH_INSTANCE_LEGACY_ENABLED = {{ horizon_launch_instance_legacy }}
 LAUNCH_INSTANCE_NG_ENABLED = {{ horizon_launch_instance_ng }}

+# A dictionary of settings which can be used to provide the default values for
+# properties found in the Launch Instance modal.
+LAUNCH_INSTANCE_DEFAULTS = {
+    'config_drive': False,
+    'enable_scheduler_hints': True,
+    'disable_image': False,
+    'disable_instance_snapshot': False,
+    'disable_volume': False,
+    'disable_volume_snapshot': False,
+    'create_volume': False,
+}
+
 # The Xen Hypervisor has the ability to set the mount point for volumes
 # attached to instances (other Hypervisors currently do not). Setting
 # can_set_mount_point to True will add the option to set the mount point
