--- /openstack/venvs/horizon-14.2.14/lib/python2.7/site-packages/openstack_dashboard/dashboards/project/static/dashboard/project/workflow/launch-instance/source/source.controller.spec.js	2018-01-15 20:44:20.000000000 +0000
+++ source.controller.spec_new-2aefe4792b37637249465733b2f8944a.js	2018-05-07 17:31:46.097703684 +0000
@@ -50,7 +50,7 @@

         scope.model = {
           allowedBootSources: [{type: 'image', label: 'Image'}],
-          newInstanceSpec: { source: [], source_type: '' },
+          newInstanceSpec: { source: [], source_type: '', create_volume_default: true },
           images: [ { id: 'image-1' }, { id: 'image-2' } ],
           imageSnapshots: [ { id: 'imageSnapshot-1' } ],
           volumes: [ { id: 'volume-1' }, { id: 'volume-2' } ],
